experiments/AMIP/coupler_driver.jl

```diff
@@ -544,7 +605,11 @@ end
 
 #=
 ## Initialize Component Model Exchange
+
+We need to ensure all initial field exchange is performed for calculating surface fluxes, before the coupled simulation is run. There are several steps to do this:
```

Comment:
could you explain this a little more? i.e. that we have to step the models before we can correctly calculate the initial surface fluxes. It's not clear to me here if "initial field exchange" is that first step of each model

Reply:
Sure. I changed this to "We need to ensure all models' initial conditions are shared to enable the coupler to calculate the first instance of surface fluxes. Some auxiliary variables (namely surface humidity and radiation fluxes) depend on initial conditions of other component models than those in which the variables are calculated, which is why we need to step these models in time and/or reinitialize them.
The concrete steps for proper initialization are:"

Reply:
Thank you!!